docs/CustomTranslator.md

```diff
@@ -111,4 +111,55 @@ class FakeTranslator {
 FakeTranslator;
 ```
 
+Example of custom translator using [Lingva (Alternative front-end for Google Translate ) API](https://github.com/thedaviddelta/lingva-translate):
+
+```js
+class LingvaTranslator {
+    // URL list of instance of LingvaTranslate, add more to use fastest one using Promise.race()
+    apiList = ["https://lingva.ml"]
+
+    translate = (text, from, to) => {
+        let promiseList = this.apiList.map(api => fetch(`${api}/api/v1/${from}/${to}/${text}`, {
+            credentials: "omit",
+            headers: {
+                "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:99.0) Gecko/20100101 Firefox/99.0",
+                Accept: "*/*",
+                "Accept-Language": "en-US,en;q=0.5",
+                "Sec-Fetch-Dest": "empty",
+                "Sec-Fetch-Mode": "cors",
+                "Sec-Fetch-Site": "same-origin"
+            },
+            method: "GET",
+            mode: "cors",
+        }))
+        return Promise.race(promiseList)
```

Comment:
Linguist may call this method frequently, we have to decrease number of requests to improve user experience, especially on mobile phones and for slow network connection.

This code may send few requests and handle only first response, so another requests useless.

Please, replace `Promise.race` to another approach. I suggest to try async loop that try to use API endpoints one by one and sort `apiList` by results.

For example, when you have `[a,b,c]` endpoints and endpoint `a` doesn't respond, but endpoint `b` works fine, we sort array to `[b,c,a]` and in next time will try `b` endpoint first.